build-scripts/android/build.sh

```diff
@@ -2,11 +2,11 @@
 
 set -e
 cd android
-./gradlew check
+./gradlew app:check
 
 if [[ $BUILD_RELEASE == 1 ]]; then
     ./gradlew assembleRelease --info --console plain | tee
 else
-    ./gradlew build --info --console plain | tee
```

Comment:
We are still linting our codebase, just not our dependencies. `./gradlew app:check` performs linting. The ./gradlew build` was also performing the same lint checks (across our app and all dependencies) so I decided to skip.

I think we should submit a PR to the problematic dependency and if/when that lands we could begin linting deps again.